src/main/java/seedu/address/logic/commands/AddTimeCommand.java

```diff
@@ -0,0 +1,207 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FREETIMETAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Birthday;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.RoomNumber;
+import seedu.address.model.person.Telegram;
+import seedu.address.model.tag.FreeTimeTag;
+
+/**
+ * Adds a free time to an existing person in the address book.
+ */
+public class AddTimeCommand extends Command {
+
+    public static final String COMMAND_WORD = "addTime";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a free time to the person identified "
```

Comment:
Perhaps you can add an example of how to use the add time command in the PR description